src/libraries/pkg/baseline/packageIndex.json

```diff
@@ -6689,6 +6707,7 @@
       "BaselineVersion": "6.0.0",
       "InboxOn": {
         "netcoreapp3.0": "4.0.4.0",
+        "netcoreapp3.1": "4.0.5.1",
```

Comment:
We should never list serviced versions here.  This has to represent what's inbox in the GA product since packages can't target a serviced release (only TFM).  A better source of these versions would be "C:\Program Files\dotnet\packs\Microsoft.NETCore.App.Ref\3.1.0\data\FrameworkList.xml", or just merge the index values from release/3.1.  cc @Anipik

Reply:
I can merge the values from https://github.com/dotnet/corefx/blob/release/3.1/pkg/Microsoft.Private.PackageBaseline/packageIndex.json, even if there was never any "netcoreapp3.0" entry. Would that suffice?

Reply:
Actually, I think you can just run a command to do this: https://github.com/dotnet/arcade/blob/6cc4c1e9e23d5e65e88a8a57216b3d91e9b3d8db/src/Microsoft.DotNet.Build.Tasks.Packaging/src/UpdatePackageIndex.cs#L167-L175
https://github.com/dotnet/arcade/blob/6cc4c1e9e23d5e65e88a8a57216b3d91e9b3d8db/src/Microsoft.DotNet.Build.Tasks.Packaging/src/build/Packaging.targets#L1296

Reply:
Here's a gist: https://gist.github.com/ericstj/3ea156677f1d679989ecd4cb21c315e8
Here's what it does: https://github.com/ericstj/runtime/commit/3889b68532f0b1639c01de96087920a0b498cf01

Reply:
Well then, time to overwrite this PR. :)